domains.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
some domains seem to be missing, would be good to see these returned but this can be done later